frontend/www/index.html

```diff
@@ -40,9 +40,9 @@ <h1 class="title">PLAYFEST</h1>
               <input type='text' id='new-playlist-input' placeholder='Name Your Playlist' ng-model='playlistName'/>
             </label>
           </div>
-          <div id='picture' ng-controller="pictureCtrl as ctrlr">
+          <div id='picture' ng-controller="pictureCtrl as picCtrlr">
             <img ng-show="imgURI !== undefined" ng-src="{{imgURI}}">
-            <button class="button button-outline button-stable" ng-click="takePicture()">TAKE PICTURE</button>
+            <button class="button button-outline button-stable" ng-click="picCtrlr.takePicture().then(ctrl.addArrayVar)">TAKE PICTURE</button>
```

Comment:
We are declaring takePicture() as a child of $scope in the controller, so should be able to call it simply as takePicture() instead of picCtrlr.takePicture().
Otherwise, we could just declare the function as this.takePicture() in the controller instead of $scope.takePicture()